packages/health-check/package.json

```diff
@@ -13,7 +13,7 @@
     "http-status-codes": "^2.2.0",
     "lodash": "^4.17.21",
     "lowdb": "^1.0.0",
-    "skynet-js": "^4.0.19-beta",
+    "skynet-js": "^4.0.25-beta",
```

Comment:
nit: We can use 26

Reply:
updated branch to master and using 4.1.0 stable